UI/src/view/point_asset/massTransitStopLayer.js

```diff
@@ -431,21 +465,36 @@ window.MassTransitStopLayer = function(map, roadCollection, mapOverlay, assetGro
   };
 
   var createNewAsset = function(coordinate, placement, stopTypes) {
+
     var default_asset_direction = {BothDirections: 2, TowardsDigitizing: 2, AgainstDigitizing: 3};
     var nearestLine = geometrycalculator.findNearestLine(excludeRoadByAdminClass(roadCollection.getRoadsForPointAssets()), coordinate.x, coordinate.y);
+    var lon, lat;
+
     if(nearestLine.end && nearestLine.start){
       var projectionOnNearestLine = geometrycalculator.nearestPointOnLine(nearestLine, coordinate);
       var bearing = geometrycalculator.getLineDirectionDegAngle(nearestLine);
+
+      if ( stopTypes !== undefined && stopTypes[0] == '7') {
```

Comment:
_.isUndefined(value)

Reply:
You mean !_.isUndefined(stopTypes)?

Reply:
@marquesfm Check modification on this code